compiler/lib/php_from_rehp.re

```diff
@@ -278,21 +279,35 @@ let topLevelIdentifiersSt = (newVarsSoFar, st) =>
   | _ => newVarsSoFar
   };
 
-let topLevelIdentifiers = (newVarsSoFar, (src, _)) =>
+let topLevelIdentifiers = (newVarsSoFar: vars, (src, _)) =>
   switch (src) {
   | Rehp.Function_declaration((id, _, _, _)) => addOne(newVarsSoFar, id)
   | Statement(stmt) => topLevelIdentifiersSt(newVarsSoFar, stmt)
   };
 
-let rec foldSources = (sourceFolder, curOut, curIn, curRevMappeds, remain) =>
+let rec foldSources =
+        (
+          sourceFolder,
+          origOut: output,
+          curOut: output,
```

Comment:
Same comment about whether or not we even need to pass `origOut`. I think we can just have `curOut`.

Reply:
I think you're right about relying on EFun to reset the labels from the source folder. I just applied the same fix to it as with the statement folder, assuming the same bug would occur.